src/app/shared/Styles.elm

```diff
@@ -0,0 +1,23 @@
+module Shared.Styles exposing (..)
+
+import Css exposing (..)
+import Html.CssHelpers
+
+
+type CssClasses
+    = Antialiased
+    | Header
+
+
+sharedNamespace =
+    Html.CssHelpers.withNamespace "shared"
+
+
+css : Stylesheet
+css =
+    stylesheet
+        [ class Antialiased
```

Comment:
So this is here but there's also `src/index.css` that includes a similar snippet but in css. Is this duplicated?